grpcurl.go

```diff
@@ -161,6 +163,37 @@ func MetadataFromHeaders(headers []string) metadata.MD {
 	return md
 }
 
+/* Expands environmental variables contained in the header string
+ * If no corresponding environmental variable is found, the header
+ * string is not changed. Hence, if the regex matches accidentally
+ * no changes are made.
+ */
+func ExpandHeaders(headers []string) []string {
+	expandedHeaders := make([]string, len(headers))
+	for idx, header := range headers {
+		if header != "" {
+			regex := regexp.MustCompile(`\${\w+}`)
```

Comment:
Definitely don't compile regex's in the loop. I know it's a simple and fast tool, so performance at this step may not usually be a concern. But we can at least hoist this out and only compile it once (make it an unexported package `var`) in case anyone ends up using this API in an unexpected way in the future.

Reply:
Done. Sorry for oversight.